### PR TITLE
Added Adequant adapter

### PR DESCRIPTION
**Pull request by relegit**
Please merge (squashed commits, as requested in #187)

### CODE REVIEW

File: src/adapters/adequant.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
+var bidfactory = require('../bidfactory.js');
+var bidmanager = require('../bidmanager.js');
+var adloader = require('../adloader.js');
+
+module.exports = function() {
+    var req_url_base = '//rex.adequant.com/rex/c2s_prebid?';
```
Comment (protonate) [inline src/adapters/adequant.js]: We are now using https in all cases so this can be hardcoded to use `https:`.

**Comment by protonate** [comment]
Thanks for the updated PR and I'd like to get this merged finally, I just need some working test params so we can confirm bids are being returned. Otherwise code and functionality looks ready to go. One note about using HTTPS in all cases so we are no longer using relative protocol syntax.

**Comment by relegit** [comment]
I've added "localhost" and "localhost.localdomain" as allowed domains, please test again

**Comment by protonate** [comment]
Ok now I'm getting a response like this:
```
var adequant_creatives = {"id":"400","seatbid":[]}
```
And this is passed to the Adequant adapter `process_bids` handler. However `seatbid` is always an empty array. Should I expect to see bids coming back? I'm requesting bids for a placement with sizes as `rightSlotSizes = [[300, 250], [300, 600]];`

It also seems that the domain restriction is to port 80, or I can't get a response using ports 8000 and up. Can this be changed to allow higher ports for testing? Running the connect server on port 80 requires admin privileges.

**Comment by relegit** [comment]
300x250 and 728x90 should be returning a result. I've lifted the port restriction, please try again

**Comment by protonate** [comment]
Thanks for the port change.

I'm still not getting any bids back in the response. Here's the request:
```
http://rex.adequant.com/rex/c2s_prebid?a=5000563&s=300x250_0.01+300x600_0.01
```

**Comment by relegit** [comment]
The server looks for the Referer header -- is it included in the query?

**Comment by mkendall07** [comment]
@relegit 
What referrer should we send to receive the test bids? If possible to whitelist `localhost` that would be helpful but we can spoof if needed.

**Comment by relegit** [comment]
I've whitelisted localhost already, it should work as long as the header is being sent over.
Would you mind emailing me your IP? I'll trace the query on my end

**Comment by relegit** [comment]
Nevermind, found the issue -- I allowed "localhost" for 728x90 size only, oops!
Please try again, should work now

**Comment by relegit** [comment]
wow, this squashing business is hairy! Think I finally got it, though!

**Comment by relegit** [comment]
merged style changes

**Comment by mkendall07** [comment]
@protonate 
Is this good to merge?

**Comment by protonate** [comment]
I'm still not getting bids back. I've tried restricting size to 728x90, referrer set to localhost over http and https. @relegit here is a gist of my request: https://gist.github.com/protonate/32cecb45285c85caddf88a4d9fe358aa

Let me know what else I can try.

**Comment by relegit** [comment]
@protonate thanks for the gist, helped me fix the problematic config :)
You should see it working now

**Comment by protonate** [comment]
Great, thank you @relegit, I'm seeing bids back in the response now.

Reviewed.

**Pull request merged**